lambda/fetch_predictions.py

```diff
@@ -0,0 +1,69 @@
+import argparse
+
+import pandas as pd
+
+from config.app import DataLakeConfig
+from precalculator.fetcher import PredictionFetcher
+
+
+def get_parameters() -> dict:
+    parser = argparse.ArgumentParser(description="Process parameters.")
+    parser.add_argument("--user", help="User ID")
+    parser.add_argument("--request", help="Request ID")
+    parser.add_argument("--model", help="Model ID")
+
+    args = parser.parse_args()
+
+    params = {"user_id": args.user, "request_id": args.request, "model_id": args.model}
+
+    return params
+
+
+def main(user_id: str, request_id: str, model_id: str) -> pd.DataFrame:
+    """Fetch predictions
+
+    Args:
+        user_id (str): _description_
+        request_id (str): _description_
+        model_id (str): _description_
+
+    Returns:
+        pd.DataFrame: _description_
+    """
+    config = DataLakeConfig()  # type: ignore
+
+    print("config", config)
+
+    fetcher = PredictionFetcher(config, user_id, request_id, model_id)
+
+    path_to_input = fetcher.get_s3_input_location()
+
+    print(path_to_input)
+
+    df_predictions = fetcher.fetch(path_to_input)
+
+    return df_predictions
+
+
+def handler(event: dict, context: dict) -> dict:
+    """Placeholder lambda function which will call main
+
+    Args:
+        event (dict): _description_
+        context (dict): _description_
+
+    Returns:
+        dict: _description_
+    """
+    return {"event": len(event), "context": len(context)}
```

Comment:
@kartikey-vyas I wonder if a link to the S3 folder containing the outputs for a particular request should be returned after this Lambda runs successfully... sending the predictions as a response over the network might not be suitable for large input files